changelogs/fragments/cron_removals.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - M(cron), removed previously deprecated C(reboot) and now requires either C(name) or C(cron_file) as unique identifier.
```

Comment:
I don't think the M(), C(), etc work here. The changelog is mainly viewed on github, I don't think it's rendered in the docs site anywhere.